08-transport.md

```diff
@@ -1,78 +1,78 @@
 # BOLT #8: Encrypted and Authenticated Transport
 
 All communications between Lightning nodes is encrypted in order to
-provide confidentiality for all transcripts between nodes, and authenticated to
-avoid malicious interference. Each node has a known long-term identifier which
+provide confidentiality for all transcripts between nodes and is authenticated in order to
+avoid malicious interference. Each node has a known long-term identifier that
 is a public key on Bitcoin's `secp256k1` curve. This long-term public key is
-used within the protocol to establish an encrypted+authenticated connection
+used within the protocol to establish an encrypted and authenticated connection
 with peers, and also to authenticate any information advertised on behalf
 of a node.
 
 ## Cryptographic Messaging Overview
 
-Prior to sending any lightning messages, nodes must first initiate the
-cryptographic session state which is used to encrypt and authenticate all
+Prior to sending any Lightning messages, nodes must first initiate the
+cryptographic session state that is used to encrypt and authenticate all
 messages sent between nodes. The initialization of this cryptographic session
 state is completely distinct from any inner protocol message header or
 conventions.
 
 The transcript between two nodes is separated into two distinct segments:
 
 1. First, before any actual data transfer, both nodes participate in an
-   authenticated key agreement protocol which is based off of the Noise
+   authenticated key agreement handshake, which is based off of the Noise
    Protocol Framework<sup>[2](#reference-2)</sup>.
-2. If the initial handshake is successful, then nodes enter the lightning
-   message exchange phase. In the lightning message exchange phase, all
-   messages are `AEAD` ciphertexts.
+2. If the initial handshake is successful, then nodes enter the Lightning
+   message exchange phase. In the Lightning message exchange phase, all
+   messages are Authenticated Encryption with Associated Data (AEAD) ciphertexts.
 
 ### Authenticated Key Agreement Handshake
 
-The handshake chosen for the authenticated key exchange is `Noise_XK`. As a
-"pre-message", we assume that the initiator knows the identity public key of
-the responder. This handshake provides a degree of identity hiding for the
-responder, its public key is _never_ transmitted during the handshake. Instead,
-authentication is achieved implicitly via a series of `ECDH` (Elliptic-Curve
-Diffie-Hellman) operations followed by a `MAC` check.
+The handshake chosen for the authenticated key exchange is `Noise_XK`. Prior
+to the handshake, the initiator must know the identity public key of
+the responder. This provides a degree of identity hiding for the
+responder, as its public key is _never_ transmitted during the handshake. Instead,
+authentication is achieved implicitly via a series of Elliptic-Curve
+Diffie-Hellman (ECDH) operations followed by a MAC check.
```

Comment:
The term "pre-message" is taken from the Noise spec, https://noiseprotocol.org/noise.html#handshake-patterns.

Reply:
Changed back to "As a pre-message" (and it reads better now with the changes to the rest of the sentence).